ts/src/main/scala/ch/epfl/ts/config/FundDistribution.scala

```diff
@@ -0,0 +1,70 @@
+package ch.epfl.ts.config
+
+/**
+ * Local money distributions.
+ * Each country is implemented as a class that inherits from FundDistribution.
+ * Statistics are entered in tuples (amount in local currency, percentage of people earning this amount).
+ * Internally, a list of integers is created that contains as many entries of
+ * a particular amount as the percentage indicates.
+ * When calling "get" a sample out of the internal list is returned using a Random object.
+ */
+
+trait FundDistribution {
+  var bins : List[Int];
+  val rand = new java.util.Random();
+  
+  def percentListToDistribution(binsInPercent: List[(Int, Double)]) : List[Int] = {
```

Comment:
Create a test suite that checks this function's output on a few simple distributions.